discovery-provider/src/api/v1/models/common.py

```diff
@@ -16,7 +16,7 @@
     "favorite",
     {
         "favorite_item_id": fields.String(required=True),
-        "favorite_type": fields.String(required=True),
+        "favorite_type": fields.String(required=True, discriminator=True),
```

Comment:
This isn't super relevant since the autogenerated doc tooling doesn't seem to understand how to use discriminators anyway (everything is much better in OAS 3.0 but unf restx doesn't support 3.0)

Especially not relevant since we're not mixing our favorites and have them separated by type.